src/amo/reducers/api.js

```diff
@@ -14,30 +14,7 @@ export const SET_USER_AGENT: 'SET_USER_AGENT' = 'SET_USER_AGENT';
 
 export const USER_AGENT_BROWSER_FIREFOX = 'Firefox';
 export const USER_AGENT_OS_ANDROID: 'Android' = 'Android';
-export const USER_AGENT_OS_BSD_DRAGONFLY: 'DragonFly' = 'DragonFly';
-export const USER_AGENT_OS_BSD_FREEBSD: 'FreeBSD' = 'FreeBSD';
-export const USER_AGENT_OS_BSD_NETBSD: 'NetBSD' = 'NetBSD';
-export const USER_AGENT_OS_BSD_OPENBSD: 'OpenBSD' = 'OpenBSD';
-export const USER_AGENT_OS_BSD_PC: 'PC-BSD' = 'PC-BSD';
 export const USER_AGENT_OS_IOS: 'iOS' = 'iOS';
-export const USER_AGENT_OS_LINUX: 'Linux' = 'Linux';
-export const USER_AGENT_OS_LINUX_ARCH: 'Arch' = 'Arch';
-export const USER_AGENT_OS_LINUX_CENTOS: 'CentOS' = 'CentOS';
-export const USER_AGENT_OS_LINUX_DEBIAN: 'Debian' = 'Debian';
-export const USER_AGENT_OS_LINUX_FEDORA: 'Fedora' = 'Fedora';
-export const USER_AGENT_OS_LINUX_GENTOO: 'Gentoo' = 'Gentoo';
-export const USER_AGENT_OS_LINUX_GNU: 'GNU' = 'GNU';
-export const USER_AGENT_OS_LINUX_LINPUS: 'Linpus' = 'Linpus';
-export const USER_AGENT_OS_LINUX_PC: 'PCLinuxOS' = 'PCLinuxOS';
-export const USER_AGENT_OS_LINUX_REDHAT: 'RedHat' = 'RedHat';
-export const USER_AGENT_OS_LINUX_SLACKWARE: 'Slackware' = 'Slackware';
-export const USER_AGENT_OS_LINUX_SUSE: 'SUSE' = 'SUSE';
-export const USER_AGENT_OS_LINUX_UBUNTU: 'Ubuntu' = 'Ubuntu';
-export const USER_AGENT_OS_LINUX_VECTOR: 'VectorLinux' = 'VectorLinux';
-export const USER_AGENT_OS_LINUX_ZENWALK: 'Zenwalk' = 'Zenwalk';
-export const USER_AGENT_OS_MAC: 'Mac OS' = 'Mac OS';
-export const USER_AGENT_OS_UNIX: 'UNIX' = 'UNIX';
-export const USER_AGENT_OS_WINDOWS: 'Windows' = 'Windows';
```

Comment:
is this unrelated tidy-up or was the File display the only place we exposed these constants?

Reply:
They were used in a different file, `src/amo/utils/index.js`, inside `userAgentOSToPlatform`, which I removed below. They weren't used anywhere else, so I removed them to make it less confusing in the future - the fewer OS-specific bits we have, the better.